### ISSUE TITLE
A's of Buttons should be one or the other

### ISSUE THREAD

**Issue by manfromjupyter** [nbgallery/nbgallery#916]
Example:
```html
a.modal-activate href="#approveChangeRequestModal" aria-haspopup="true" data-toggle="modal" tabindex="-1"
            button.btn.btn-success Approve Change Request
```
Here the styling works but screen readers jumping by tabs would not read it has a popup.

Need to research how to do things like these. Sometimes we want it to be a link but look like a button or be a button and be triggerable.

**Comment by mcrutch** [comment]
Can the button have a data-toggle/data-target?

**Comment by manfromjupyter** [comment]
Yes by the looks of it https://getbootstrap.com/docs/3.4/javascript/#:~:text=primary%20btn%2Dlg%22-,data%2Dtoggle%3D%22modal%22%20data%2Dtarget,-%3D%22%23myModal%22
